string codigo_tarea
string[] vector_mover
string[] vector_girar
string[] vector_x_y
---
string feedback
int64  eval
